LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-z-index-fractions-001-expected.txt

```diff
@@ -1,4 +1,4 @@
 
 PASS testing z-index: calc(3 / 2)
-FAIL testing z-index: calc(-3 / 2) assert_equals: expected "-1" but got "-2"
+PASS testing z-index: calc(-3 / 2)
```

Comment:
Seems like this isn't quite enough test coverage for all the boundaries and edge cases.